mgmt/src/tests/nat.rs

```diff
@@ -441,25 +438,23 @@ mod tests {
 
     #[test]
     #[traced_test]
-    #[cfg(any())]
     fn test_full_config() {
         let mut config = build_sample_config();
         config.validate().expect("Failed to validate config");
-        //config
-        //  .build_internal_config() //BROKEN
-        //.expect("Failed to build internal config");
-        println!("Internal config: {:#?}", config.internal);
-        let nat_tables = config
+        let internal = build_internal_config(&config).expect("Failed to build internal config");
+        println!("Internal config: {:#?}", &internal);
+        config.set_internal_config(internal);
+        let nat_tables = &config
             .internal
-            .expect("Failed to build internal config")
+            .unwrap()
             .nat_table
             .expect("Failed to build NAT tables");
 
         let (mut nat, mut tablesw) = StatelessNat::new("stateless-nat");
-        tablesw.update_nat_tables(nat_tables);
+        tablesw.update_nat_tables(nat_tables.clone());
 
         // Template for other packets
-        let pt = build_test_ipv4_packet(u8::MAX).unwrap();
+        let _pt = build_test_ipv4_packet(u8::MAX).unwrap();
```

Comment:
I think this line can be removed completely, it seems to be a leftover from when I wrote the tests. Packet build has moved to `check_packet()`.